recipes/pcapplusplus/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "PcapPlusPlus" %}
+{% set version = "22.05" %}
+{% set sha256 = "5f299c4503bf5d3c29f82b8d876a19be7dea29c2aadcb52f2f3b394846c21da9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/seladb/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+  host:
+  - libpcap
+  run:
+  - libpcap
```

`libpcap` has a `run_exports`: https://github.com/conda-forge/libpcap-feedstock/blob/8587d4c6155e9d5dd3c549cb6282c8d36e50c5bc/recipe/meta.yaml#L21

Thus this will automatically be added here, so please omit the explicit specification.
```suggestion
  run:
  - libpcap
```

---

Can you remove this, please? Then we can merge.
```suggestion
```